backend/src/main/java/com/woowacourse/moragora/application/UserService.java

```diff
@@ -109,6 +113,17 @@ public void delete(final UserDeleteRequest request, final Long id) {
         userRepository.delete(user);
     }
 
+    private void validateUserExists(final String email) {
+        final Optional<User> user = userRepository.findByEmail(email);
```

Comment:
```suggestion
        final Optional<User> user = userRepository.findByEmailAndProvider(email,Provider.CHECKMATE);
```
이렇게 바꿔볼 수 있을 거 같아요! 이렇게 되면 User 객체의 `isProviderCheckmate()`도 사라질 수 있겠네요!

Reply:
좋네요~! 적용했습니다!!